crates/core/src/client/mod.rs

```diff
@@ -59,26 +64,29 @@ use crate::models::ConversionError;
 
 pub struct KakarotClient<P: Provider + Send + Sync> {
     starknet_provider: Arc<P>,
+    starknet_account: SingleOwnerAccount<Arc<P>, LocalWallet>,
```

Comment:
can we rename this to `deployer_account`?

Reply:
done